### ISSUE TITLE
Implementing STLs in C++ under data structures

### ISSUE THREAD

**Issue by Pratyaksha462** [OpenGenus/cosmos#6422]
**This is a(n):**

- [x] New algorithm
- [ ] Update to an existing algorithm
- [ ] Error
- [ ] Proposal to the Repository

**Details:**

I am adding the folder for implementing the STLs functions in C++ . This would help the programmer to get the template of the code.
All in this I would be adding all the necessary information needed to implement these libraries. This Folder will come under the Data Structure because these are the data structures that would be easily implemented. Also the implementation will be ready-to-use templates.

**Comment by Pratyaksha462** [comment]
This issue  is made new because there is no information of the issue that was two years ago.

**Comment by manish831** [comment]
I want to do work upon this issues, plz assign it to me.

**Comment by AdiChat** [comment]
@manish831 yes, as this involve multiple algorithms, you can work on a part in parallel.

**Comment by HridoyHazard** [comment]
i also want to contribute in this.

**Comment by naveenbisht31** [comment]
Hey, I would like to contribute as well. Please assign it to me.

**Comment by Haricharan0310** [comment]
Hey @AdiChat I would like to Contribute to this Issue as well, Could you please assign it to me?

**Comment by prashantkr29** [comment]
Hey @AdiChat I would like to contribute as well. Could you please assign it to me?

**Comment by naveenbisht31** [comment]
@AdiChat where we need to upload code?

**Comment by Coderayush13** [comment]
I wanted to contribute in this project .

**Comment by devanshiii26** [comment]
can you please assign to me?

**Comment by vidhi0511** [comment]
Hi, I am pretty confident that I can do it. Please assign this task to me.

**Comment by Denisthatorf** [comment]
I have a very nice implementation of std::function. Can I contribute to this one?

**Comment by Purvi-sai** [comment]
I would like to contribute in this issue, Please assign this to me

**Comment by Purvi-sai** [comment]
I would like to contribute in this issue, Please assign this to me
